src/binary-reader.cc

```diff
@@ -391,6 +397,14 @@ Result BinaryReader::ReadInitExpr(Index index, bool require_i32) {
       break;
     }
 
+    case Opcode::V128Const: {
+      v128 value_bits;
```

Comment:
Since SIMD is a future feature, we want to only allow it if the feature flag is enabled. See other examples using the `ERROR_UNLESS_OPCODE_ENABLED` macro in this file.

Reply:
Agree.